src/impact-dashboard/FacilityInformation.tsx

```diff
@@ -238,6 +268,17 @@ const AgeGroupRow: React.FC<AgeGroupRowProps> = (props) => {
     }
   }
 
+  const checkAgeConflict = (model: object) => {
+    if (Date.now() > props.warnedAt + 10000) {
+      if (includesUnknownAges(model) && includesKnownAges(model)) {
+        addToast(
+          "To prevent duplicate counting, known and unknown ages cannot be entered together. Please clear either the unknown age input or all age inputs.",
+        );
```

Comment:
should this be an error toast?

Reply:
@jessex should I make this toast red/error colored?

Reply:
Agreed, let's make this toast red!